docs/rename_function.py

```diff
@@ -0,0 +1,74 @@
+"""
+This script requires developers to add the following information:
+1. add file and directory name to srcdir_srcfiles
+2. add file and function name to srcfiles_srcfuncs
+3. add expected display name for the function to display_names
+"""
+
+
+
+import os
+import itertools
+from shutil import copyfile
+
+# Add the function names with the src file
+# srcfiles_srcfuncs = { srcfile : [func1, func2..]}
+# key : srcfile
+# values per key : function names that should be changed
+# If the file is already present, just add the func name in the respective values
+# Create new entry if the srcfile is not present
+srcfiles_srcfuncs = {
+    "hpat_pandas_series_functions.py" : ["hpat_pandas_series_append", "hpat_pandas_series_ne", "hpat_pandas_series_iloc"]
+}
+
+
+# Add the filenames and the parent directory

```

Comment:
Maybe is better to use Python style for variable documentation? It looks particular the same as for function but below the variable.

Reply:
Do you mean the variable names should be changed here?

Reply:
I meant just Python style for variable __doc__ string. It is not necessary but it might be better to get.

Reply:
Yes sure. I will do that. I just wanted to clear that I understood your comment correctly.